setup.py

```diff
@@ -31,6 +31,10 @@
         setuptools = None
         from distutils.core import Extension
         from distutils.core import setup
+    try:
+        from wheel.bdist_wheel import bdist_wheel
+    except ImportError:
+        bdist_wheel = None
```

Comment:
I wonder: is it possible (or worth it) to raise in case we're explicitly running cbuildwheels tool? Something like (not tested, just to illustrate what I mean):

```python
    try:
        from wheel.bdist_wheel import bdist_wheel
    except ImportError:
        if "RUNNING_CIBUILDWHEEL_CLI" in os.environ:
            raise
```
....or perhaps `cbuildwheel` command already fails if `bdist_wheel` package is not installed?